Arriba/Arriba/Model/Table.cs

```diff
@@ -156,6 +156,22 @@ public ICollection<ColumnDetails> ColumnDetails
             }
         }
 
+        public ICollection<string> ColumnNames
```

Comment:
I don't think this is necessary.  There is already a ColumnDetails property.  Can just get the Name field from there.  Look how prepare changed for SelectQuery.